### PR TITLE
Docusaurus upgrade debugging

### PR DESCRIPTION
**Pull request by donghwipark**
1. docusaurus-plugin-sass"from  0.1.08 to 0.1.11
2. config modification
3. Implement Navbar/index.js accordingly

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -13,15 +13,18 @@ module.exports = {
   projectName: 'gulpjs.github.io',
   // The theme
   themeConfig: {
-    disableDarkMode: true,
+    colorMode: {
+      defaultMode: 'light',
+      disableSwitch: false,
```
Comment (phated) [inline docusaurus.config.js:18]:
```suggestion
      disableSwitch: true,
```

This should be true, as our theme doesn't work with dark mode.

**Review by phated — CHANGES_REQUESTED** [review]
I reviewed the site and found a bunch of issues that need to be fixed. I will comment on the PR thread.

**Comment by phated** [comment]
The algolia search box changed and needs styling updates to match our search box:

The changes make it look like this:
Screen Shot 2021-01-27 at 12 40 21 PM

But it should look like this:
Screen Shot 2021-01-27 at 12 43 09 PM

And on mobile:
Screen Shot 2021-01-27 at 12 49 12 PM

But should look like:
Screen Shot 2021-01-27 at 12 46 53 PM

I'm okay adjusting to to a dark-background search icon for mobile, but it would need to come with additional navbar updates to make things look proportionate.

**Comment by phated** [comment]
The word "logo" appears in the footer for some reason now:

Screen Shot 2021-01-27 at 12 46 09 PM

**Comment by phated** [comment]
The plugins page keyword lists got messed up:
Screen Shot 2021-01-27 at 12 45 41 PM
Screen Shot 2021-01-27 at 12 45 47 PM

I assume this was a change with the underlying css library that we use from docusaurus

**Comment by phated** [comment]
We should also remove this patch https://github.com/gulpjs/gulpjs.github.io/pull/130 that was put into place to fix an issue until we upgraded past alpha.66

**Comment by phated** [comment]
@donghwipark let me know once you've completed those fixes and I will review again.

**Comment by donghwipark** [comment]
@phated I fixed the issues you mentioned above. Please confirm.

File: src/css/docs.css [hunk 2]
```diff
@@ -167,11 +171,27 @@ table tr td + td + td:last-child {
...
+.DocSearch.DocSearch-Button:hover {
+  background-color: #9a2829;
+  color: #ffffff;
+  height: 2.25rem;
+}
+.navbar__items .DocSearch-Button .DocSearch-Search-Icon {
+  width: 1rem;
+  height: 1rem;
+  color: #ffffff;
+}
```
Comment (phated) [inline src/css/docs.css:192]: There's a bunch of issues here, but I'm just going to push a fix.

File: src/css/docs.css [hunk 3]
```diff
@@ -188,8 +208,21 @@ table tr td + td + td:last-child {
...
+    height: 2.25rem;
+    font-size: 1rem;
+    border-radius: 0;
+    color: #ffffff;
+    background-color: #9a2829;
+    font-family: var(--ifm-font-family-base);
+  }
+  span.DocSearch-Button-Placeholder {
+    display: block;
+  }
```
Comment (phated) [inline src/css/docs.css:225]: Same as above. Also, you added code instead of replacing the navbar__search stuff,

File: src/css/docs.css [hunk 4]
```diff
@@ -289,18 +321,18 @@ table tr td + td + td:last-child {
...
   box-shadow: 6px 6px 0 0 var(--ifm-color-secondary);
-  transition: box-shadow .1s ease-in, transform .1s ease-in;
+  transition: box-shadow 0.1s ease-in, transform 0.1s ease-in;
 }
 
 /* Fix: show "Copy to clipboard" button */
 button.copyButton_node_modules-\@docusaurus-theme-classic-src-theme-CodeBlock- {
-	color: var(--ifm-color-white);
+  color: var(--ifm-color-white);
 }
```
Comment (phated) [inline src/css/docs.css:338]: This was supposed to be removed as I mentioned. Will push the update.

File: src/theme/Navbar/index.js [hunk 5]
```diff
@@ -14,11 +14,10 @@ import isInternalUrl from '@docusaurus/isInternalUrl';
 import { useLocation } from '@docusaurus/router';
 
 import SearchBar from '@theme/SearchBar';
-import Toggle from '@theme/Toggle';
+// import Toggle from '@theme/Toggle';
```
Comment (phated) [inline src/theme/Navbar/index.js:17]: Why was this code commented out? Just delete it if that's your solution. I'll push the fix.

File: src/theme/Navbar/index.js [hunk 6]
```diff
@@ -134,11 +133,11 @@ function Navbar() {
...
-  const { logoLink, logoLinkProps, logoImageUrl, logoAlt } = useLogo();
-  const onToggleChange = useCallback(
-    e => (e.target.checked ? setDarkTheme() : setLightTheme()),
-    [setLightTheme, setDarkTheme],
-  );
+
+  // const onToggleChange = useCallback(
+  //   e => (e.target.checked ? setDarkTheme() : setLightTheme()),
+  //   [setLightTheme, setDarkTheme],
+  // );
```
Comment (phated) [inline src/theme/Navbar/index.js:140]: Same, and you should have cleaned up the other code that used these values. I'll push.

File: src/theme/Navbar/index.js [hunk 7]
```diff
@@ -149,36 +148,30 @@ function Navbar() {
...
-          {!disableDarkMode && (
+          {/* {!disableDarkMode && (
             <Toggle
               className={styles.displayOnlyInLargeViewport}
               aria-label="Dark mode toggle"
               checked={isDarkTheme}
               onChange={onToggleChange}
             />
-          )}
+          )} */}
```
Comment (phated) [inline src/theme/Navbar/index.js:174]: Same as above. Don't comment code, just delete it. I'll push.

File: src/css/docs.css (hunk 2 above)
Comment (phated) [reply src/css/docs.css:192]: Specifically, you should have been using the css variables and reviewing the different states, hover/active/etc.

**Review by phated — COMMENTED** [review]
@donghwipark great work on this! I have some cleanup that I'm going to push then I'll merge!

**Pull request merged**